mainnet-dryrun-2/gentx/gentx-Whispernode.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"WhisperNode🤐","identity":"9C7571030BEF5157","website":"https://REStake.WhisperNode.com","security_contact":"security@whispernode.com","details":"WhisperNode operates robust, high up-time validators across the Cosmos ecosystem. Auto-compound your rewards with our REStake app. Join our Discord: https://discord.gg/4E5KZsRtjE / tweet @ https://twitter.com/WhisperNode 🐦"},"commission":{"rate":"0.100000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"archway1x7uajx8e5j98yyptld90yf8crcewk3f6fxvqrx","validator_address":"archwayvaloper1x7uajx8e5j98yyptld90yf8crcewk3f6fz9zw0","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"arUQy+g4jWlpaJl8TgT5TPZZX1cm4zwyeueu7tYhEV8="},"value":{"denom":"aconst","amount":"50000000000000"}}],"memo":"9ddb8770de202cf59a389a6f93c751f68c5d3ecc@141.95.157.139:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A9zoGcJ3Qb4hFeS858DDWVWmrNVni7Cbu5xkXw9lRG5S"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["d9v8bx0RISkRn198AYzTgWpknEYzNQd+YnLfRt5CAzla9m92aZg6CgMsutR4IfdIYFsDyeflMxoSU/d1BjhBgw=="]}
```

Comment:
Gentx fee amount is empty
Check docs https://docs.archway.io/validators/becoming-a-validator/running-a-validator-node#create-your-validator